features/list_categories.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Visitor views list of categorized products
+  As a visitor,
+  In order to choose product more easily
+  I would like to see the products sorted in categories
+
+  Background:
+    Given the following categories exist
+      | title    |
+      | Starters |
+      | Entrees  |
+      | Desserts | 
+    Given the following products exist
```

Comment:
Please change to `And the following products exist`

Reply:
When I remove the category table from the Background, this one will be *Given* again